src/WebJobs.Script.WebHost/WebHostServiceCollectionExtensions.cs

```diff
@@ -222,7 +236,7 @@ private static void AddLinuxContainerServices(this IServiceCollection services)
                 //todo: Replace with legion specific service
                 if (environment.IsAnyLinuxConsumption())
                 {
-                    var instanceManager = s.GetService<IInstanceManager>();
+                    var instanceManager = s.GetService<LinuxInstanceManager>();
```

Comment:
After the change above back to `IInstanceManager`, make sure to undo this change.